BrainPortal/lib/cbrain_task_generators/templates/task_params.html.erb.erb

```diff
@@ -420,7 +423,7 @@
     <%- end -%>
         )
   <%- end -%>
-  <%- if param['description'] -%>
```

Comment:
This will remove the descriptions of all other inputs if the decriptor is a single_file scenario. Not what you wanted.